packages/contracts/package.json

```diff
@@ -7,14 +7,20 @@
     "clean": "rm -rf types",
     "build": "pnpm clean && pnpm compile && pnpm types",
     "compile": "forge build --force",
-    "types": "typechain --target ethers-v5 \"out/**/!(*.t|test|Test|Script|Vm|console|console2).sol/!(*.abi).json\" --out-dir types && tsc"
+    "types": "typechain --target ethers-v5 \"out/**/!(*.t|test|Test|Script|Vm|console|console2).sol/!(*.abi).json\" --out-dir types && tsc",
+    "prettier": "prettier --write \"src/**/*.sol\" --plugin=prettier-plugin-solidity",
+    "solhint:check": "solhint --config ./.solhint.json \"src/**/*.sol\"",
+    "solhint": "pnpm solhint:check --fix"
   },
   "devDependencies": {
     "@ethersproject/abi": "^5.0.0",
     "@ethersproject/bytes": "^5.0.0",
     "@ethersproject/providers": "^5.0.0",
     "@typechain/ethers-v5": "^10.0.0",
     "ethers": "^5.5.2",
+    "prettier-plugin-solidity": "^1.0.0-beta.19",
```

Comment:
Any idea if this also runs `solhint --fix` (like prettier + eslint)?

Reply:
I don't think it does – that would be nice though, I'm going to work some more on this PR and I will investigate how we might run `solhint --fix` in VS Code automatically. I'm going to convert this to a draft PR in the meantime.